openedx_webhooks/tasks/pr_tracking.py

```diff
@@ -227,6 +226,10 @@ def desired_support_state(pr: PrDict) -> PrDesiredInfo:
     else:
         state = "closed"
 
+    # A label of jira:xyz means we want a Jira issue in the xyz Jira.
+    label_names = set(lbl["name"] for lbl in pr["labels"])
+    desired.jira_nicks = {name.partition(":")[-1] for name in label_names if name.startswith("jira:")}
+
     desired.jira_initial_status = "Needs Triage"
```

Comment:
This is an assumption about what states are expected in any given Jira project that is added. We should probably document this somewhere.  ie.  "Setting up a Jira Project to work with openedx-webhooks".  I'm also okay with something small in the README for now.

Reply:
As it turns out, we never use this value at the moment.  We'll get issues created, and then fix up the initial status somehow.  Ideally, we wouldn't have to specify the initial status, the project could just set it implicitly.

Reply:
I've deleted it.